doc/user/tutorials/sistr/index.md

```diff
@@ -19,62 +19,79 @@ Initial Data
 
 The data for this tutorial comes from <https://irida.corefacility.ca/downloads/data/irida-sample-data.zip>. It is assumed the sequence files in `miseq-run-salmonella/` have been uploaded into appropriate samples as described in the [Web Upload Tutorial][]. Before starting this tutorial you should have a project with samples that appear as follows:
 
-![tutorial-sistr-samples.png][]
+![sistr-samples]
 
 Adding Samples to the Cart
 ==========================
 
 Before a pipeline can be run a set of samples and sequence read data must be selected and added to the cart. For this tutorial please select the single sample and click the **Add to Cart** button.
 
-![add-to-cart.png][]
+![add-to-cart]
 
 Once the samples have been added to the cart, the samples can be reviewed by clicking on the **Cart** button at the top.
 
-![cart-button.png][]
+![cart-button]
 
 Selecting a Pipeline
 ====================
 
-Once inside the cart all available pipelines will be listed in the main area of the page.
+Once inside the cart all available pipelines will be listed in the main area of the page. For this tutorial, we will select the **SISTR Pipeline**.
 
-![select-a-pipeline-view.png][]
-
-For this tutorial, we will select the **SISTR Pipeline**.
+![pipeline-select]
 
 Selecting Parameters
 ====================
 
-Once the pipeline is selected, the next page provides an overview of all the input files, as well as the option to modify parameters.
+Once the pipeline is selected, the next page provides an overview of all the input files, as well as the option to modify parameters. We will use the default parameters.
 
-![sistr-pipeline-page.png][]
+![sistr-pipeline-page]
 
-We will use the default parameters. Please use the **Ready to Launch?** button to start the pipeline.
+Please use the **Ready to Launch?** button to start the pipeline.
 
-![ready-to-launch-button.png][]
+![launch-button]
 
 Once the button is selected you should see a screen showing that your pipeline has been launched.
 
-![pipeline-launch.png][]
+![pipeline-launched]
 
 Monitoring Pipeline Status
```

Comment:
Something I'm not seeing anywhere is the pipeline duration for running pipelines.  I thought we were going to jam it in the status bar.  Is it somewhere else?
![image](https://user-images.githubusercontent.com/8951708/76865279-79d5e000-6830-11ea-8f9d-e6decb945d73.png)

Reply:
Added duration underneath the steps. It was just displayed on the Settings > Details tab before